src/i18n/locales/ko-KR/settings.json

```
@@ -1,6 +1,7 @@
 {
 	"zoom": {
 		"level": "줌 레벨",
+		"customScale": "커스텀 줌",
```

Biome (2.4.15)

[error] The key customScale was already declared.

(lint/suspicious/noDuplicateObjectKeys)